manifests/integrations/tcp_check.pp

```diff
@@ -0,0 +1,129 @@
+# Class: datadog_agent::integrations::tcp_check
+#
+# This class will install the necessary config to hook the tcp_check in the agent
+#
+# Parameters:
+#   name
+#       (Required) - Name of the service. 
+#        This will be included as a tag: instance:<name>.
+#
+#   host
+#       (Required) - Host to be checked. 
+#        This will be included as a tag: url:<host>:<port>.
+#
+#   port
+#       (Required) - Port to be checked. 
+#        This will be included as a tag: url:<host>:<port>.
+#
+#   timeout
+#       (Optional) - Timeout for the check. Defaults to 10 seconds.
+#
+#   threshold
+#       (Optional) - Used in conjunction with window. An alert will 
+#        trigger if the check fails <threshold> times in <window> attempts.
+#
+#   window
+#       (Optional) - Refer to �threshold�.
```

Comment:
Interesting copy/paste/edit from the documentation on the check, oops. those quotes snuck through. I assume this was the "non-utf8" error, which was bugging me! haha I will sort this out, thanks for the heads up.